WordPress/src/main/java/org/wordpress/android/ui/media/services/MediaUploadService.java

```diff
@@ -35,46 +34,8 @@
 
 public class MediaUploadService extends Service {
     public static final String SITE_KEY = "mediaSite";
```

Comment:
Nitpicking: These two `*_KEY` variables can be made private.  Thanks to the Code inspector ;)